### PR TITLE
QA dashboard should display links to CI jobs for smoketests (stage, production) if such jobs exists

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

File: dashboard/src/smoke_tests.py [hunk 1]
```diff
@@ -28,6 +38,19 @@ def read_logs(self):
...
+    def read_ci_results(self):
+        """Read results generated on CI side."""
+        if self._ci_jobs is not None:
+            for system, job_name in SmokeTests.CI_JOBS.items():
+                if job_name is not None:
+                    # the job has been setup -> let's check its status on CI
+                    url = self._ci_jobs.get_job_url(system, job_name)
+                    name = self._ci_jobs.get_job_name(system, job_name)
+                    job_status = self._all_jobs_statuses.get(name)
+                    print(url, name, job_status)
```
Comment (jpopelka) [inline dashboard/src/smoke_tests.py]: left-over ?

Comment (tisnik) [reply dashboard/src/smoke_tests.py]: ah, definitely. Sorry for that :/

**Comment by tisnik** [comment]
[test]

**Comment by tisnik** [comment]
[test]

**Comment by tisnik** [comment]
[test]

**Pull request merged**